content/en/continuous_testing/explorer/_index.md

```diff
@@ -16,44 +16,69 @@ further_reading:
 - link: "/continuous_testing/explorer/search/"
   tag: "Documentation"
   text: "Learn how to search through your test batches"
-  
+- link: "/continuous_testing/explorer/search_runs/"
+  tag: "Documentation"
+  text: "Learn how to search through your test runs"
 ---
 
 ## Overview
 
-The [Continuous Testing Explorer][1] provides visibility into the CI/CD jobs executing your Continuous Testing tests. 
+The [Explorer][1] provides visibility into all test runs and CI batches for **Synthetic Monitoring** and **Continuous Testing**. 
 
 {{< img src="continuous_testing/explorer_ci_batches.png" alt="Continuous Testing Explorer" style="width:100%;">}}
 
 You can accomplish the following actions:
 
-* Investigate every CI/CD job execution to identify and troubleshoot failing test results
-* Compare test results executed against various devices and browsers to pinpoint cross browser and device issues
-* Keep track of the progress of your tests in CI/CD pipelines
-* Identify flaky tests to fix
+* Compare test runs executed against various devices and browsers to pinpoint cross browser and device issues
+* Examine performance issues with result timing facets and filter runs by failing status codes
```

Comment:
> * Examine performance issues with _result timing facets_ and filter runs by failing status codes

Once I found this in the UI, I understood what it was, but it's hard to know without context. Maybe a screenshot would help, if you were so inclined!